tpl/template_funcs.go

```diff
@@ -1317,6 +1317,17 @@ func ModBool(a, b interface{}) (bool, error) {
 	return res == int64(0), nil
 }
 
+func FormatNum(format string, num interface{}) (string, error) {
+	switch conv := num.(type) {
+	case int:
+		return renderInteger(format, conv)
+	case float64:
+		return renderFloat(format, conv)
+	default:
+		return "", fmt.Errorf("Invalid type to format.")
+	}
```

Comment:
Couple questions:  shouldn't this switch handle all int and float types? And why have `renderInteger` when you just work with floats anyway?  Just use a single func and do the type casting here.

```go
switch conv := num.(type) {
case int, int8, int16, int32, int64, float32, float64:
    return renderNumber(format, conv.(float64))
default:
    return "", fmt.Errorf("Invalid type to format.")
}
```